terraform_deploy/tig.tf

```diff
@@ -1,5 +1,5 @@
 module "tig" {
-  source = "source will be override by override.py"
+  source = "https://github.com/podaac/tig/releases/download/0.11.0/tig-terraform-0.11.0.zip"
```

Comment:
shouldn't this be 0.12.0?

Reply:
well there the override file so its whatever we put in there when we generate it in the github actions, terrafrom 1.5.3 made it so we have to put in a real url so i just put in 11